backend/project_name/settings/local_base.py

```diff
@@ -18,6 +18,8 @@
 
 AUTH_PASSWORD_VALIDATORS = []  # allow easy passwords only on local
 
+REDIS_URL = "redis://result:6379"
```

Comment:
Better to put in https://github.com/vintasoftware/django-react-boilerplate/blob/main/backend/.env.example
To avoid having a hardcoded value here

Reply:
Also, if it's necessary to have this to run the project, update the README accordingly (I think on the part that says how to run w/o docker).